dymos/trajectory/trajectory.py

```diff
@@ -314,13 +320,24 @@ def _setup_linkages(self):
 
                 if options['connected']:
                     if var2 == 'time':
-                        phase2.set_time_options(input_initial=True)
+                        phase2.set_time_options(input_initial=True, fix_initial=False)
                     elif var2 == '*':
-                        phase2.set_time_options(input_initial=True)
+                        phase2.set_time_options(input_initial=True, fix_initial=False)
                         for state_name in phase2.state_options:
-                            phase2.set_state_options(state_name, conected_initial=True)
+                            if isinstance(phase2, AnalyticPhase):
+                                try:
+                                    phase2.set_state_options(state_name, input_initial=True, fix_initial=_unspecified)
```

Comment:
Won't an AnalyticPhase always raise the NotImplmentedError with input_initial=True? Could cut out the middleman and just raise the runtimeerror.
Also, be  careful. Maybe adding the conditional here was unintentional, since you don't add it in the other part of the branch.

Reply:
The different conditional was unintentional yes.
My IDE gripes if I use a different signature for the method in a derived class, but I've decided to omit any arguments that pertain to the state as a design variable.
This particular method now just raises RuntimeError if `phase2` is an `AnalyticPhase`.